api/internal/builtins/AnnotationsTransformer.go

```diff
@@ -1,6 +1,4 @@
 // Code generated by pluginator on AnnotationsTransformer; DO NOT EDIT.
-// pluginator {(devel)  unknown   }
```

Comment:
Remove pluginator's version line.
Since we're developing in the same repo, it should always be devel.